kin-devplatform-sdk/src/main/java/kin/devplatform/base/IBasePresenter.java

```diff
@@ -6,5 +6,7 @@
 
 	void onDetach();
 
+	void onClose();
```

Comment:
What is the reason to create a different method instead of using the provided `onDetach()`?

Reply:
during the lifetime of a presenter it can be attached and detached repeatedly but it is only closed once.

Reply:
some things should not be cleaned up during detach but only during close.